### ISSUE TITLE
Error when running runbot.bat

### ISSUE THREAD

**Issue by ghost** [Just-Some-Bots/MusicBot#206]
Getting an error when I log out of the bot and then try and run my bat file? Any ideas as to why? It's probably something simple but I suck at this sort of thing!

https://gyazo.com/8a4643c17c94538461e37b542cfa88b9

Link to the error message.

**Comment by imayhaveborkedit** [comment]
I'm going to assume you had python previously installed for something, but never had pip installed with that version.  The wiki tutorial has you install python with pip, and surely you followed that.  You can rerun the python installer and modify the install to add pip, then the bot should work.

**Comment by Stackee** [comment]
[Screenshot](http://prntscr.com/atnlmw)
I'm getting this same error. I installed according to the tutorial (doesn't really mention pip but I checked both those boxes at the start like it says to). I then followed imayhaveborkedit's advice and reran the python installer - pip was clearly checked in the modify section, it updated almost instantly and said complete but still get the same error again. 

Wondering if maybe it matters which drive I'm running it from? I think Python is on my SSD where as the MusicBot is on my HDD.
